src/test/java/org/example/c03/controlFlow/FactorPrinterTest.java

```diff
@@ -0,0 +1,29 @@
+package org.example.c03.controlFlow;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class FactorPrinterTest {
+    @Test
+    public void given10ShouldReturn12510AsDivisors() {
+        int number = 10;
+        int[] myArray = {1, 2, 5, 10};
```

Comment:
Rename myArray to expectedResult for all tests in this class.

Reply:
done